man/systemd-run.xml

```diff
@@ -221,12 +221,31 @@
 
         <listitem><para>When invoking the command, the transient service connects its standard input and output to the
         terminal <command>systemd-run</command> is invoked on, via a pseudo TTY device. This allows running binaries
-        that expect interactive user input as services, such as interactive command shells.</para>
+        that expect interactive user input/input as services, such as interactive command shells.</para>
```

Comment:
What does this slash mean? It's unclear.

Reply:
oops, that's a typo. was supposed to say "input/output", as in "I/O"